<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWindow</class>
 <widget class="QWidget" name="StatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Production Stats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">

   <!-- Encabezados -->
   <item row="0" column="0">
    <widget class="QLabel" name="lblStationHeader">
     <property name="text">
      <string>Station</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblProcessedHeader">
     <property name="text">
      <string>Processed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblBufferHeader">
     <property name="text">
      <string>Buffer Size</string>
     </property>
    </widget>
   </item>

   <!-- Fila 1: Assembler-1 -->
   <item row="1" column="0">
    <widget class="QLabel" name="stationName1">
     <property name="text">
      <string>Assembler-1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLCDNumber" name="lcdProcessed1">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLCDNumber" name="lcdBuffer1">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

   <!-- Fila 2: Assembler-2 -->
   <item row="2" column="0">
    <widget class="QLabel" name="stationName2">
     <property name="text">
      <string>Assembler-2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLCDNumber" name="lcdProcessed2">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLCDNumber" name="lcdBuffer2">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

   <!-- Fila 3: Tester -->
   <item row="3" column="0">
    <widget class="QLabel" name="stationName3">
     <property name="text">
      <string>Tester</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="lcdProcessed3">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLCDNumber" name="lcdBuffer3">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

   <!-- Fila 4: QualityControl -->
   <item row="4" column="0">
    <widget class="QLabel" name="stationName4">
     <property name="text">
      <string>QualityControl</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLCDNumber" name="lcdProcessed4">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLCDNumber" name="lcdBuffer4">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

   <!-- Fila 5: Packer -->
   <item row="5" column="0">
    <widget class="QLabel" name="stationName5">
     <property name="text">
      <string>Packer</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLCDNumber" name="lcdProcessed5">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLCDNumber" name="lcdBuffer5">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

   <!-- Fila 6: Shipping -->
   <item row="6" column="0">
    <widget class="QLabel" name="stationName6">
     <property name="text">
      <string>Shipping</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLCDNumber" name="lcdProcessed6">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLCDNumber" name="lcdBuffer6">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>

